addon/package.json

```diff
@@ -65,7 +65,7 @@
     "@ember-data/serializer": ">= 3.28",
     "@ember-data/store": ">= 3.28",
     "@ember/string": ">= 3.0.0",
-    "@ember/test-helpers": ">= 2.4.2",
+    "@ember/test-helpers": "^2.9.3 || >= 3",
```

Comment:
This could be considered breaking, but I don't think this will be an issue for people.

Latest v2 supports later versions of ember etc etc and has lots of backported fixes.